.github/workflows/ci.yml

```
@@ -0,0 +1,72 @@
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup node env
+        uses: actions/setup-node@v2.1.2
+
+      - name: Install dependencies Frontend
+        run: npm ci
+
+      - name: Cypress run
+        uses: cypress-io/github-action@v6
+        
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)